src/core/config/resolvers/imports.ts

```diff
@@ -15,7 +15,7 @@ export async function resolveImportsOptions(options: NitroOptions) {
   options.imports.presets.push(...getNitroImportsPreset());
 
   // Add h3 auto imports preset
-  const h3Exports = await resolveModuleExportNames("h3", {
+  const h3Exports: string[] = await resolveModuleExportNames("h3", {
```

Comment:
Type is already inferred: 

<img width="441" alt="image" src="https://github.com/user-attachments/assets/c0bb9263-4e97-475d-bca9-4a456ede70e6">

Reply:
oh. i see. thanks.